libraries/aws_iam_policy.rb

```diff
@@ -85,8 +85,8 @@ def has_statement?(criteria = {})
       actions    = s[:Action] || []
       notactions = s[:NotAction] || []
       effect     = s[:Effect]
-      resource   = s[:Resource]
-      statement_match = true if (criteria[:NotAction].nil? ? actions.include?(criteria[:Action]) : notactions.include?(criteria[:NotAction])) && effect.eql?(criteria[:Effect]) && resource.eql?(criteria[:Resource])
+      resource   = s[:Resource].sort
```

Comment:
Thanks for this contribution @rgevaert !  I had a look at this and came across the following; if we have a policy such as e.g. 
```
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Effect": "Allow",
            "Action": "*",
            "Resource": "*"
        }
    ]
}
```
The above would result in the statement being something like this e.g. 
```
{:Effect=>"Allow", :Action=>"*", :Resource=>"*"}
```
So it means `s[:Resource]` is a string and not sortable.  The same is true for the passed in criteria field if we try testing something similar like:
```
    it { should have_statement('Effect' => 'Allow', 'Resource' => '*', 'Action' => '*') }
```
Could you add some protection such that the `sort` is only attempted when an array is encountered for example?

Reply:
Sure, did that now.

Reply:
Thanks, looks good!